sh_scrapy/middlewares.py

```diff
@@ -60,6 +60,8 @@ def process_request(self, request, spider):
             request.meta[HS_PARENT_ID_KEY] = request_id
 
     def process_response(self, request, response, spider):
+        if type(response).__name__ == "DummyResponse":
```

Comment:
Can we add a comment linking to https://github.com/scrapy-plugins/scrapy-zyte-api/issues/112 ? This would help provide context in the future and newcomers.